src/instances/billing.php

```diff
@@ -52,4 +52,6 @@
 $DBLIB->where("users_userid",  $AUTH->data['instance']['instances_billingUser']);
 $PAGEDATA['billingUser'] = $DBLIB->getOne("users", ["users.users_name1", "users.users_name2", "users.users_userid"]);
 
+$PAGEDATA['billingEnabled'] = $CONFIG['STRIPE_KEY'] !== null;
```

Comment:
Just to flag with the new config not everything is loaded into that array. Stripe key is a `"specialRequest" => true` variable so it is not stored in memory for every single page. Instead to access it you have to use `$CONFIGCLASS->get('STRIPE_KEY')`

Reply:
(resolving this, but just an fyi I didn't document anywhere else @cherry-john)